src/data/polynomial/degree/reverse_mul.lean

```diff
@@ -0,0 +1,605 @@
+/-
+Copyright (c) 2020 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+
+import tactic
+import data.polynomial.degree.basic
+import data.polynomial.degree.trailing_degree
+
+open polynomial finsupp finset
+
+namespace polynomial
+
+variables {R : Type*} [semiring R] {f : polynomial R}
+
+lemma nonempty_of_card_nonzero {α : Type*} {s : finset α} : s.card ≠ 0 → s.nonempty :=
```

Comment:
This is very similar to `finset.card_pos`, and several other lemmas on `data.finset.basic`. Could you either just use those lemmas, or make this an `iff` and put it next to those other lemmas?